pkg/datarepair/checker/checker_test.go

```diff
@@ -150,8 +150,9 @@ func TestIdentifyIrreparableSegments(t *testing.T) {
 				RemotePieces: pieces,
 			},
 		}
-		// put test pointer to db
-		metainfo = planet.Satellites[0].Metainfo.Service
+		// update test pointer in db
+		err = metainfo.Delete(ctx, "fake-piece-id")
```

Comment:
Out of curiosity, why wasn't it needed before?

Reply:
Because the test wants to replace the pointer. Now `metainfo.Put` fails if a pointer for this path already exists. Hence, the test needs to execute Delete and Put.

Reply:
oh, yes silly question, I've read that later in the review and I didn't think to review my previous comments before submitting it.